e2e/_suites/fleet/fleet.go

```diff
@@ -1035,11 +1047,9 @@ func (fts *FleetTestSuite) anAttemptToEnrollANewAgentFails() error {
 
 	agentInstaller := fts.getInstaller()
 
-	profile := agentInstaller.Profile // name of the runtime dependencies compose file
-
-	containerName := fmt.Sprintf("%s_%s_%s_%d", profile, fts.Image+"-systemd", common.ElasticAgentServiceName, 2) // name of the new container
+	containerName := fts.getContainerName(agentInstaller, 2) // name of the new container
```

Comment:
The index is not being utilized here so getContainerName is always returning `_1`